src/features/search/index.tsx

```diff
@@ -41,37 +41,35 @@ const DateRowLayoutContainer = styled.div`
     }
 `
 
-
 type Search = (query: string) => void;
 
 type StockListItem = {
     symbol: string,
     name: string
 }
 
-
 const socket = socketService.get();
 
 export const Search: FC = () => {
 
     const dispatch = useDispatch();
 
-    const [query, setQuery] = useState<string>('Apple Inc (AAPL)');
+    const [query, setQuery] = useState('Apple Inc (AAPL)');
     const [stockList, setStockList] = useState<StockListItem[]>([])
-    const [isOpen, toggleIsOpen] = useState<boolean>(false)
+    const [isOpen, toggleIsOpen] = useState(false)
     const dropSelect = useRef<HTMLDivElement>(null)
     const inputSelect = useRef<HTMLInputElement>(null)
-    const [selectedStock, setSelectedStock] = useState<string[]>(['Apple Inc', '(AAPL)'])
+    const [selectedStock, setSelectedStock] = useState(['Apple Inc', '(AAPL)'])
 
-    const tags: string[] = useSelector(({ companyOverview: { tags }}: AppState) => tags);
-    const primaryExchange: string | null = useSelector(({ keyStats: { primaryExchange } }: AppState) => primaryExchange);
-    const isUSMarketOpen: boolean = useSelector(({ keyStats: { isUSMarketOpen } }: AppState) => isUSMarketOpen)
-    const price: PriceSingleDataPoint = useSelector((store: AppState) => {
+    const tags = useSelector((state: AppState) => state.companyOverview.tags);
+    const primaryExchange = useSelector((state: AppState) => state.keyStats.primaryExchange);
+    const isUSMarketOpen = useSelector((state: AppState) => state.keyStats.isUSMarketOpen)
```

Comment:
nit: Could you not combine your multiple useSelectors into one? Example:

`const { primaryExchange, isUSMarketOpen, latestTime etc } = useSelector((state: AppState) => state.keyStats)`

Reply:
Yep will pull relevant items into single useSelector and push to this PR before merging

Reply:
pushed changes